### DISCUSSION TITLE
how to build on Fedora

### DISCUSSION THREAD

**Discussion by lersek** [Frederick888/external-editor-revived#76] [Q&A] [answered]
Hi Frederick,

thank you for reanimating `exteditor`!

How can I build this (i.e., the messaging host) on Fedora from source?

In particular I don´t want `cargo` to automatically download a tree of dependencies from wherever. If possible I'd like to satisfy those dependencies from Fedora-provided packages, or at least understand what each individual dependency is and where it comes from. I'd especially like to avoid a deeply recursive dependency tree.

Thank you,
Laszlo

**Comment by Frederick888** [comment, marked as answer]
> How can I build this (i.e., the messaging host) on Fedora from source?

Wiki. (I should probably set up a bot at this point...)

> [...] satisfy those dependencies from Fedora-provided packages

It's certainly a surprise to me that Fedora does this, which can be of quite some hassle (and futility tbbh). But anyway, I don't use Fedora so my answer is simply: no idea. You should reach out to Fedora's community especially considering it does such a peculiar thing when packaging Rust applications.

**Comment by lersek** [comment]
Thanks. I had certainly checked the wiki before asking my question. Unfortunately, I overlooked the "Source installation" article https://github.com/Frederick888/external-editor-revived/wiki/Source-Installation . Found it now.
